### PR TITLE
Aligned domain event naming across terraform, json and c-sharp code

### PR DESCRIPTION
**Pull request by jonasdmoeller**
Alignment of domain event naming across Terraform, JSON and C# code.

### CODE REVIEW

File: build/infrastructure/main/sbt-charges-domain-events.tf [hunk 1]
```diff
@@ -41,7 +41,7 @@ module "sbts_charges_charge_command_accepted_publish" {
   }
 }
 
-module "sbts_charges_charge_accepted_dataavailable" {
+module "sbts_charges_charge_information_operations_accepted_dataavailable" {
```
Comment (prtandrup) [inline build/infrastructure/main/sbt-charges-domain-events.tf:44]: @Mech0z wasn't there somthing about a 35 character limit here?

Comment (jonasdmoeller) [reply build/infrastructure/main/sbt-charges-domain-events.tf]: fixed

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/local.settings.sample.json [hunk 2]
```diff
@@ -54,35 +54,35 @@
...
-    "CHARGE_PRICE_COMMAND_RECEIVED_SUBSCRIPTION_NAME": "sbtsub-charges-charge-price-command-received",
+    "CHARGE_PRICE_COMMAND_RECEIVED_SUBSCRIPTION_NAME": "sbts_charges_price_command_received",
 
     // Internal, charge, accepted
-    "CHARGE_INFORMATION_OPERATIONS_ACCEPTED_DATAAVAILABLE_SUBSCRIPTIONNAME": "sbtsub-charges-charge-accepted-dataavailable",
-    "CHARGE_INFORMATION_OPERATIONS_ACCEPTED_SUBSCRIPTION_NAME": "sbtsub-charges-charge-command-accepted",
-    "CHARGE_INFORMATION_OPERATIONS_ACCEPTED_PUBLISH_SUBSCRIPTION_NAME": "sbtsub-charges-charge-command-accepted-publish",
+    "CHARGE_INFORMATION_OPERATIONS_ACCEPTED_DATAAVAILABLE_SUBSCRIPTION_NAME": "sbts_charges_info_operations_accepted_da",
+    "CHARGE_INFORMATION_OPERATIONS_ACCEPTED_SUBSCRIPTION_NAME": "sbts_charges_info_operations_accepted_publish",
+    "CHARGE_INFORMATION_OPERATIONS_ACCEPTED_PUBLISH_SUBSCRIPTION_NAME": "sbts_charges_info_operations_accepted",
```
Comment (prtandrup) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/local.settings.sample.json:65]: These two lines seems to have been switched around, value-wise?

**Review by prtandrup — APPROVED** [review]
Approved with a single minor comment

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/local.settings.sample.json (hunk 2 above)
Comment (jonasdmoeller) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/local.settings.sample.json]: fixed

**Pull request merged**